docs/user/exercises/modeling.rst

```diff
@@ -1,2 +1,224 @@
 Modeling exercise
-=================
\ No newline at end of file
+=================
+.. contents:: Content of this document
+    :local:
+    :depth: 3
+
+
+Overview
+--------
+
+Conducting a Modeling exercise consists of 3 steps:
+
+1. **Instructor prepares exercise:** Creates and configures the modeling exercise in Artemis.
+2. **Student solves exercise:** Student works on the exercise and submits the solution.
+3. **Instructor assesses submissions:** Reviews the submitted exercises and creates results for the students.
+
+Setup
+-----
+
+The following sections describe the supported features and the process of creating a new modeling exercise.
+
+- Open |course-management|
+- Navigate into **Exercises** of your preferred course
+
+    .. figure:: modeling/course-management-course-dashboard.png
+              :align: center
+
+Create new modeling exercise
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Click on **Create new modeling exercise**
+
+    .. figure:: modeling/create-new-modeling-exercise.png
+              :align: center
+
+The following screenshot illustrates the first section of the form. It consists of:
+
+- **Title**: Title of an exercise
+- **Categories**: Category of an exercise
+- **Difficulty**: Difficulty of an exercise. (No level, Easy, Medium or Hard)
+- **Mode**: Solving mode of an exercise. *This cannot be changed afterwards* (Individual or Team)
+- **Release Date**: Date after which students can access the exercise
+- **Due Date**: Date till when students can access the exercise
+- **Assessment Due Date**: Date till when instructors can assess the exercise submitted by students
```

Comment:
Technically the instructors can assess the exercise even after the Assessment Due Date.
I'd prefer, `Date after which students can view the feedback of the assessments from the instructors` or something else